tools-local/tasks/mobile.tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -257,7 +257,7 @@ private static string GetLatestBuildTools(string androidSdkDir)
         string? buildTools = Directory.GetDirectories(Path.Combine(androidSdkDir, "build-tools"))
             .Select(Path.GetFileName)
             .Where(file => !file!.Contains("-"))
-            .Select(file => Version.TryParse(Path.GetFileName(file), out Version? version) ? version : default)
+            .Select(file => Version.TryParse(Path.GetFileName(file), out Version? version) ? (version ?? default) : default)
```

Comment:
```suggestion
            .Select(file => { Version.TryParse(Path.GetFileName(file), out Version? version); return version; })
```
More straightforward, less verbose way to say the same.